tests/manager/manager-ctor-read_preference-error-002.phpt

```diff
@@ -1,42 +1,42 @@
 --TEST--
-MongoDB\Driver\Manager::__construct(): invalid read preference (maxStalenessMS)
+MongoDB\Driver\Manager::__construct(): invalid read preference (maxStalenesSeconds)
 --FILE--
 <?php
 
 require_once __DIR__ . '/../utils/tools.php';
 
 echo throws(function() {
-    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?maxstalenessms=1231');
+    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?maxstalenessseconds=1231');
 }, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
 
 echo throws(function() {
-    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?maxStalenessMS=1231');
+    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?maxStalenessSeconds=1231');
 }, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
 
 echo throws(function() {
-    $manager = new MongoDB\Driver\Manager(null, ['maxstalenessms' => 1231]);
+    $manager = new MongoDB\Driver\Manager(null, ['maxstalenessseconds' => 1231]);
 }, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
 
 echo throws(function() {
-    $manager = new MongoDB\Driver\Manager(null, ['maxStalenessMS' => 1231]);
+    $manager = new MongoDB\Driver\Manager(null, ['maxStalenessSeconds' => 1231]);
 }, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
 
 echo throws(function() {
-    $manager = new MongoDB\Driver\Manager(null, ['readPreference' => 'secondary', 'maxStalenessMS' => -1]);
+    $manager = new MongoDB\Driver\Manager(null, ['readPreference' => 'secondary', 'maxStalenessSeconds' => -2]);
```

Comment:
Let's add test cases for `0` (another boundary case) and `42` (something less than 90) as well.